code/drasil-lang/lib/Language/Drasil/Chunk/Eq.hs

```diff
@@ -74,6 +74,7 @@ instance TypeChecks (QDefinition Expr) Expr Space where
   -- "normal" way does not work for Functions because it leaves function input
   -- parameters left unchecked. It's probably preferred to be doing type
   -- checking at time of chunk creation rather than here, really.
+  typeCheckExpr (QD q [] e) = pure (sy q $= e, Boolean)
```

Comment:
I had to explicitly make this a `sy q` if the functional inputs are empty. I think we should push the operation to `apply` however. I will create a ticket and fix this tomorrow.

Reply:
It is indeed odd - one shouldn't be applying to 0 arguments.

Reply:
I agree, I will fix it in #3111.